assets/js/modules/adsense/util/is-zero-report.test.js

```diff
@@ -21,6 +21,80 @@
  */
 import { isZeroReport } from './is-zero-report';
 
+const validReport = {
+	totalMatchedRows: '1',
+	headers: [
+		{
+			name: 'TOTAL_EARNINGS',
+			type: 'METRIC_CURRENCY',
+			currencyCode: 'EUR',
+		},
+		{
+			name: 'IMPRESSIONS',
+			type: 'METRIC_TALLY',
+		},
+		{
+			name: 'PAGE_VIEWS_RPM',
+			type: 'METRIC_CURRENCY',
+			currencyCode: 'EUR',
+		},
+	],
+	rows: [
+		{
+			cells: [
+				{
+					value: '0.00',
```

Comment:
Here's an example of a real report that returns `0.00` for earnings when requesting `TOTAL_EARNINGS`. See the warning below. This is still non-zero report though because it has impressions.